adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -120,6 +121,17 @@ var AdyenCheckoutHybris = (function () {
                 }
             }
 
+            if (paymentMethod === "afterpay_default") {
+                if (!this.afterPay.state.isValid) {
```

Comment:
Not sure if this is already happening, but it's nice to show the fields in red on submit if they're not filled in. You might want to check if it's triggering that behaviour, if not it should work if we call `this.afterPay.showValidation()` before this `isValid` check.
This is a tip given by Marc from checkout when I asked about credit cards, but he said it should work for all components.

Reply:
Right now it only marks red if field was selected but not filled. I will add this.

Reply:
added it.